src/rpc/account.ts

```diff
@@ -169,6 +172,39 @@ export class AccountRpc {
         });
     }
 
+    public sendParcel(params: {
+        parcel: Parcel;
+        account: PlatformAddress | string;
+        passphrase?: string;
+    }): Promise<{ hash: H256; nonce: U256 }> {
+        const { parcel, account, passphrase } = params;
+        if (!PlatformAddress.check(account)) {
+            throw Error(
+                `Expected the second argument to be a PlatformAddress value but found ${account}`
```

Comment:
the second argument -> account parameter, since it isn't a positional argument.

Reply:
done